recipes/data/testData.js

```diff
@@ -0,0 +1,65 @@
+const recipe1 = {
```

Comment:
FYI In the TK repo we use `fabricators` to generate test data

Reply:
👍 
I was pressed for time on the tests - I couldn't TDD it as I didn't know what I was going to do at the start.
😆